src/Parsers/ASTAlterQuery.h

```diff
@@ -222,10 +222,16 @@ class ASTAlterCommand : public IAST
 
     ASTPtr clone() const override;
 
+    // This function is only meant to be called during application startup
+    // For reasons see https://github.com/ClickHouse/ClickHouse/pull/59532
+    static void setFormatAlterCommandsWithParentheses(bool value) { format_alter_commands_with_parentheses = value; }
```

Comment:
Should we call it in `clickhouse-client`  and `clickhouse-local` as well? They print a formatted query:

```
localhost.localdomain :) SeLeCt NuLl

SELECT NULL

Query id: fdbaa7b8-75c0-4f58-b79c-46b9ae290e1e

┌─NULL─┐
│ ᴺᵁᴸᴸ │
└──────┘

1 row in set. Elapsed: 0.005 sec. 
```

I think it's fine to unconditionally set it to `true` for that purpose